README.md

```diff
@@ -841,12 +841,13 @@ Libraries to help manage database schemas and migrations.
 * [Takeout](https://github.com/tighten/takeout) - A Docker-based development-only dependency manager.
 * [Vagrant](https://www.vagrantup.com/) - A portable development environment utility.
 
-### Virtual Machines
-*Alternative PHP virtual machines.*
+### Virtual Machines and Compilers
+*Alternative PHP virtual machines and compilers to translate PHP into other programming languages.*
```

Comment:
> ... to translate PHP into other programming languages

Is this part necessary?

Maybe "Alternative PHP virtual machines and compilers" would suffice?

Not all compilers are source-to-source compilers.

Reply:
Yes, you're right. I changed it to your suggestion.
Anyway, waiting for approval from @ziadoz